docs/source/peer_ledger_snapshot.md

```diff
@@ -0,0 +1,164 @@
+* **Transactions IDs**
+  * This consists of the transaction IDs that have been used in the channel until the last block in the snapshot.
+* **Collection config history**
+  * This contains the history of the collection configurations for all chaincodes. Recall from our documentation on private data that the collection configuration derives the endorsement and dissemination policies for private data collections.
+
+In addition, the snapshot contains two metadata files that help the data in the snapshot to be verified. This snapshot metadata file is expected to be same across snapshots of a channel for a particular height.
+
+One of these files contains a JSON record called `_snapshot_signable_metadata.json` with the following fields:
+
+* `channel_name`: the name of the channel.
+* `last_block_number`: the block height when the snapshot was taken.
+* `last_block_hash`: a hash of the block at the height the snapshot was taken.
+* `previous_block_hash`: a hash of the block prior to the `last_block`.
+* `state_db_type` (the value of this field will be either CouchDB or SimpleKeyValueDB (also known as LevelDB).
+* `snapshot_files_raw_hashes`, is a JSON record that contains the hashes of the files above.
+
+This another metadata file is a JSON record with the following two fields:
+
+* `snapshot_hash`, the hash of the file `_snapshot_signable_metadata.json` and can be treated as a hash of the snapshot.
+* `last_block_commit_hash`, which is included if the snapshot generating peer is equipped to compute the block commit hashes.
+
+Note that the file types explained here is a superset of all of the files that might be included in a snapshot. If admins find some of these file types missing in their snapshots (for example, the collection config history) this is not mean the snapshot is incomplete. The channel might not have any collections.
+
+## Joining a channel using a snapshot
+
+When joining a channel using the genesis block, a command similar to `peer channel join --blockpath mychannel.block` is issued. When joining the peer to the channel using a snapshot, issue a command similar to:
+
+```
+peer channel joinbysnapshot --snapshotPath <path to snapshot>
+```
+
+To verify that the peer has joined the channel successfully, issue a command similar to:
+
+```
+peer channel getinfo -c <name of channel joined by snapshot>
+```
+
+Additionally, if the peer has not already installed a chaincode being used on the channel, do so, and then issue a query. A successful return of data indicates that the peer has successfully joined using the snapshot. You can then install all of the chaincodes being used on the channel. If the snapshot is used by a new organization, and the channel contains the definitions for the chaincodes that are defined using the [new chaincode lifecycle](../chaincode_lifecycle.html), the new organization will need to approve the definition of these chaincodes can be invoked using the peer.
```

Comment:
```suggestion
Additionally, if the peer has not already installed a chaincode being used on the channel, do so, and then issue a query. A successful return of data indicates that the peer has successfully joined using the snapshot. You can then install all of the chaincodes being used on the channel. If the snapshot is used by a new organization, and the channel contains the definitions for the chaincodes that are defined using the [new chaincode lifecycle](../chaincode_lifecycle.html), the new organization will need to approve the definition of these chaincodes before they can be invoked on the peer.
```

Actually, this is not just for new organizations, you must approveformyorg for all chaincodes on each bootstrapped channel, as that is the step that associates a certain chaincode package with the channel definition on that peer.

Reply:
If the organization is already on the channel, it would have already gone through the "approveformyorg" step, right? This does seem like something only a "new" organization would have to do.

Reply:
@joealewine You're right, it was late at night... only new orgs need to approveformyorg.